spec/s3.spec.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
```

Comment:
Would you move this to `spec/storage/s3.spec.js` to mirror the layout in `lib/`?